src/js/options.ts

```diff
@@ -121,18 +121,30 @@ export interface BaseReactNativeOptions {
    * @default 2
    */
   appHangTimeoutInterval?: number;
+
+  /**
+   * The max queue size for capping the number of envelopes waiting to be sent by Transport.
+   */
+  maxQueueSize?: number;
+}
+
+export interface ReactNativeTransportOptions extends BrowserTransportOptions {
+  /**
+   * @deprecated use `maxQueueSize` in te root of the SDK options.
+   *
+   * Setting this option does not have any effect.
+  */
```

Comment:
H: Not sure if this comment makes sense because the code does:

> const maxQueueSize = passedOptions.maxQueueSize
>     // eslint-disable-next-line deprecation/deprecation
>     ?? passedOptions.transportOptions?.bufferSize
>     ?? DEFAULT_OPTIONS.maxQueueSize;

So it does have an effect, right?